debian/ubuntu-advantage-tools.postinst

```diff
@@ -265,6 +265,24 @@ mark_reboot_cmds_as_needed() {
     add_notice "$msg_name"
 }
 
+check_wrong_fips_metapackage_on_cloud() {
+      fips_metapkg="ubuntu-fips"
+
+      cloud_id=""
+      if which "cloud-id" > /dev/null ; then
+        cloud_id=$(cloud-id)
+      fi
+
+      # If the package is not installed, we don't want the postinst script to fail
+      fips_installed=$(dpkg-query -W --showformat='${Status}\n' $fips_metapkg 2>/dev/null || true)
```

Comment:
Might you want "${db:Status-Status}" instead of  ${Status} since we don't have to support this on Trusty?  db:Status-Status gives us a simple "installed" to check for instead of "install ok installed"